onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -7699,12 +7699,23 @@ def aten_softmax_no_dtype(self: TFloatOrBFloat16, dim: int) -> TFloatOrBFloat16:
     return result
 
 
+@torch_op("aten::sort", traceable=True)
 def aten_sort(
-    self: TensorType, dim: int = -1, descending: bool = False
-) -> tuple[TensorType, TensorType]:
-    """sort(Tensor self, int dim=-1, bool descending=False) -> (Tensor values, Tensor indices)"""
+    self: TReal, dim: int = -1, descending: bool = False, stable: bool = False
+) -> tuple[TReal, INT64]:
+    """sort(Tensor self, int dim=-1, bool descending=False, bool stable=False) -> (Tensor values, Tensor indices)"""
 
-    raise NotImplementedError()
+    self_is_scalar = IsScalar(self)
```

Comment:
if self is scalar, is it possible to just return self? e.g. op.Identity(self) ?

Reply:
Had tried that initially, but looks like we cannot return from inside a control-flow block

Reply:
Should it be made traceonly? The only control-flow logic is rank-dependent, and we are anyway assuming that is known as export-time. Then, you can do what Justin suggests.

Reply:
Alternatively, you can assign the output of identity to result, then return result outside of the control flow. But trace only makes sense